app/frontend/styles/components/information_summary.scss

```diff
@@ -12,7 +12,7 @@
 .govuk-info-summary__body {
```

Comment:
These `.scss` files will get deleted at some point, `flashes.scss` is where notification styling lives